lib/airbrake/rake/tasks.rb

```diff
@@ -1,67 +1,42 @@
+require 'airbrake-ruby'
+
 namespace :airbrake do
   desc 'Verify your gem installation by sending a test exception'
-  task test: :environment do
-    require 'pp'
-
-    begin
-      raise StandardError, 'Exception from the test Rake task'
-    rescue => ex
-      response = Airbrake.notify_sync(ex)
+  task test: (:environment if defined?(Rails)) do
+    if Airbrake[:default].is_a?(Airbrake::NilNotifier)
+      raise 'The Airbrake library is loaded but not configured'
     end
 
-    notifiers = Airbrake.instance_variable_get(:@notifiers).map do |name, notif|
-      cfg = notif.instance_variable_get(:@config)
-      filters = notif.instance_variable_get(:@filter_chain)
-      "#{name}:\n  " + [cfg, filters].pretty_inspect
-    end.join("\n")
-
-    if !response
-      puts <<-NORESPONSE.gsub(/^\s+\|/, '')
-        |Couldn't send a test exception. There are two reasons for this:
-        |
-        |1. Airbrake ignored this exception due to misconfigured filters
-        |2. Airbrake was configured to ignore the '#{Rails.env}' environment.
-        |   To fix this try one of the following:
-        |     * specify another environment via RAILS_ENV
-        |     * temporarily unignore the '#{Rails.env}' environment
-      NORESPONSE
-    elsif response['error']
-      puts <<-ERROR.gsub(/^\s+\|/, '')
-        |Error occurred: #{response['error']}
-        |Make sure that your Project ID and Project Key are correct:
-        |https://github.com/airbrake/airbrake-ruby#project_id--project_key
-      ERROR
+    require 'pp'
+
+    response = Airbrake.notify_sync('Exception from the test Rake task')
+    raise response['error'] if response['error']
```

Comment:
AFAIK it is `response['message']` and you still need to check `response['code']` or `response['type']`